code/modules/mod/mod_control.dm

```diff
@@ -142,36 +142,35 @@
 	RegisterSignal(src, COMSIG_SPEED_POTION_APPLIED, .proc/on_potion)
 	movedelay = CONFIG_GET(number/movedelay/run_delay)
 
+
+/obj/item/mod/control/handle_atom_del(atom/deleting_atom)
```

Comment:
what if the parts are outside us, on the human wearing them for example

Reply:
We were already calling QDEL_LIST(mod_parts), but it is best be certain they're null'd. Resolved.